pyproject.toml

```diff
@@ -10,12 +10,14 @@ repository = "https://github.com/mbhall88/tbpore"
 keywords = ["tuberculosis", "nanopore", "diagnostics", "genomics", "variant-calling", "resistance-prediction"]
 
 [tool.poetry.scripts]
-snakefmt = 'tbpore.tbpore:main'
+tbpore = 'tbpore.tbpore:main'
 
 [tool.poetry.dependencies]
 python = ">=3.8,<4.0"
-click = "^8.0.3"
+click = "8.0.3"
```

Comment:
why did you remove the `^`?

Reply:
I didn't find what `^` mean, but I think means install a higher version? Anyway, with `^` we were installing `click` at a higher version than `8.0.3` (don't remember exactly which one). The [`click` version was incompatible with our `black` version](https://github.com/psf/black/issues/2964), so [I fixed `click` to `8.0.3`](https://github.com/mbhall88/tbpore/pull/8/commits/684912e2adf10773df4abce008e48c9550527c6a), and things worked. If you prefer, we can instead force `black` version to be `>=22.3.0`, which has a fix for this issue, and the `click` can be any version. I personally prefer to fix the version of every dependency to a specific version, as updates can break tools/other dependencies, but I will leave the choice to you

Reply:
https://python-poetry.org/docs/dependency-specification/#caret-requirements

I think leave click with the caret and remove and version pinning for black.

Reply:
Done in https://github.com/mbhall88/tbpore/pull/8/commits/78619f6f87562c7d159a0420c42b6cc355c8a8e5 . Is it fine keep `black = ">=22.3.0"` to ensure it is compatible with recent `click` versions?